orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/kato/tasks/DisableInstancesTask.groovy

```diff
@@ -29,18 +29,24 @@ import org.springframework.stereotype.Component
 @Component
 class DisableInstancesTask implements Task {
 
-  @Autowired KatoService katoService
+  @Autowired
+  KatoService katoService
 
   @Override
   TaskResult execute(Stage stage) {
-    def actions = [[disableInstancesInDiscoveryDescription: stage.context], [deregisterInstancesFromLoadBalancerDescription: stage.context]]
-    def taskId = katoService.requestOperations(actions)
+    def actions = [[disableInstancesInDiscovery: stage.context]]
+    def healthProviders = ["Discovery"]
+    if (stage.context.cloudProvider != 'titus') {
```

Comment:
I know we've got a one or two of these `!= 'titus'` blocks ... but should we use a constant for slightly improved traceability?

or

If `CloudProvider.supportsLoadBalancer(stage.context.cloudProvider)`? (with a default of true). Then at least we have these decisions centralized somewhere.